app/views/smart_proxies/plugins/_openscap.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= javascript 'foreman_openscap/openscap_proxy' %>
+<div class="row">
+  <h3><%= feature %></h3>
+</div>
+<%= show_feature_version(feature.downcase) %>
+<div class="row">
+  <% msg = _("Proxy failed to send a report from spool to Foreman. This indicates a corrupted report format. Report has been moved to :corrupted_dir on proxy for inspection.") %>
```

Comment:
I'd change `Report has been moved to :corrupted_dir on proxy for inspection` to `Report has been moved to directory for storing corrupted files on proxy for later inspection.`